src/components/colorbar/draw.js

```diff
@@ -10,6 +10,7 @@
 'use strict';
 
 var d3 = require('d3');
+var tc = require('tinycolor2');
```

Comment:
Yeah, you're right:

![image](https://cloud.githubusercontent.com/assets/6675409/14565071/5ff525da-02f7-11e6-9f03-90e6bcb76a90.png)

`var tinycolor = require('tinycolor2');` is the winner by a big margin though:"


![image](https://cloud.githubusercontent.com/assets/6675409/14565074/65f61f02-02f7-11e6-96e9-4f87cc4393ef.png)

Wanna change the other `var tc` too?